session/xmlrpc.go

```diff
@@ -126,14 +126,12 @@ func (x *XmlRpcTransport) DoRequest(
 	}
 
 	mask := options.Mask
+
 	if mask != "" {
-		if !strings.HasPrefix(mask, "mask[") && !strings.Contains(mask, ";") && strings.Contains(mask, ",") {
+		if !strings.HasPrefix(mask, "mask[")  {
```

Comment:
I presume the old mask format is not supported anymore

Reply:
They still work in this string format. 
```xml
<member><name>SoftLayer_ObjectMask</name><value><struct><member><name>mask</name><value><string>mask[itemCategory;id]</string></value></member></struct></value></member>
```

Its possible to send in an XML structure for the objectMask, but it is a lot easier to just send in a string like this since this project doesn't have any fancy objectMask logic to worry about.